<?xml version="1.0"?>
<launch>
  <arg name="task_kind" default="1" />
  <arg name="rqt_gui" default="False" />
  <!-- bringup -->
  <include file="$(find jsk_uav_forest_common)/launch/m100_bringup.launch" >
  </include>

  <!-- Odometry -->
  <node pkg="jsk_uav_forest_common" type="odom_generator.py" name="odom_gen" output="screen" >
    <arg name="use_lidar" value="True" />
    <arg name="use_guidance_vel" value="True" />
    <arg name="lidar_tc" value="0.5" />
    <arg name="lidar_noise_cut_thresh" value="0.5" />
    <arg name="guidance_vel_weight" value="0.9" />
  </node>

  <!-- perception + motion -->
  <include file="$(find jsk_uav_forest_motion)/launch/motion.launch">
    <arg name="use_dji" value="True" />
    <arg name="uav_odom_topic_name" value="/modified_odom" /> 
    <arg name="task_kind" value="$(arg task_kind)" />
    <arg name="rqt_gui" value="$(arg rqt_gui)" />
  </include>
</launch>
